webui/react/src/components/CreateUserModal.tsx

```diff
@@ -109,7 +109,7 @@ const CreateUserModalComponent: React.FC<Props> = ({ onClose, user, viewOnly }:
             (await removeRolesFromUser({ roleIds: Array.from(rolesToRemove), userId: user.id }));
         }
         fetchUserRoles();
-        if (currentUser && currentUser.id === user.id) checkAuth();
+        if (currentUser && currentUser.id === user.id) await checkAuth();
```

Comment:
This call is at the end of the block and doesn't throw, so `await`ing this shouldn't have any effect:
```suggestion
        if (currentUser && currentUser.id === user.id) checkAuth();
```

Reply:
Reverted this change.